Assets/Scripts/Game Controllers/WinConditionController.cs

```diff
@@ -0,0 +1,17 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class WinConditionController : MonoBehaviour
```

Comment:
I think `WinConditionController.cs` should be renamed to `LevelEndTriggerController.cs`, and I also think that `Win Condition.prefab` should be renamed to `Level End Trigger.prefab`. The current names are fine with me though.